test_local.py

```
@@ -0,0 +1,148 @@
+            assert False, "Should have raised ValueError for missing env vars"
+        except ValueError as e:
+            print(f"✅ Correctly caught missing env vars: {e}")
+        
+        # Restore env vars
+        os.environ.update(original_env)
+        
+        # Test invalid rate_per_km
+        os.environ['rate_per_km'] = 'invalid'
+        try:
+            validate_environment()
+            assert False, "Should have raised ValueError for invalid rate_per_km"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)